client/sites/staging-site/components/staging-site-sync-modal/index.tsx

```diff
@@ -463,6 +489,19 @@ export default function SyncModal( {
 								/>
 							</VStack>
 						) }
+						<HStack alignment="left" spacing={ 1 }>
+							<Text color="var(--studio-gray-40)">
+								{ displayBackupDate
+									? createInterpolateElement( __( 'Backup contents from: <date />' ), {
+											date: <span>{ displayBackupDate }</span>,
+									  } )
+									: __( 'There are no backups' ) }{ ' ' }
```

Comment:
Nit: A typo

```suggestion
									: __( 'There are no backups.' ) }{ ' ' }
```

Also, maybe we could display the site slug, or just write the source here to clarify it a bit further, like "There are no backups of the staging/production site yet."